plugins/bbcode/plugin.js

```diff
@@ -157,6 +157,12 @@
 					// as "span" with an attribute marker.
 					if ( part == 'email' || part == 'img' )
 						attribs.bbcode = part;
+                                            
+                                        //special handling of left, right and center tags
```

Comment:
Code style is not consistent with rest of the code. Comment should start uppercased, there should be a space etc.